src/mono-runtimes/mono-runtimes.targets

```diff
@@ -280,7 +280,7 @@
           Include="@(_MonoRuntime->'$(_MSBuildDir)\lib\%(Identity)\%(OutputMonoPosixHelperFilename).d.%(NativeLibraryExtension)')"
       />
       <_RuntimeEglibHeaderSource
-          Include="@(_MonoRuntime->'$(MonoSourceFullPath)\sdks\builds\android-%(Identity)-$(_MonoSdksConfiguration)\mono\eglib\eglib-config.h')"
+          Include="@(_MonoRuntime->'$(MonoSourceFullPath)\sdks\builds\android-%(Identity)-$(_MonoSdksConfiguration)\share\mono-2.0\mono\eglib\eglib-config.h')"
```

Comment:
This didn't work either; apparently things are not consistent.

Current [**macOS PR Pipeline Release Build**](https://jenkins.mono-project.com/job/xamarin-android-pr-pipeline-release/125/) fails with:

```
…/src/mono-runtimes/mono-runtimes.targets(397,5): error MSB3030: Could not copy the file "…/xamarin-android/external/mono/sdks/builds/android-x86-release/share/mono-2.0/mono/eglib/eglib-config.h" because it was not found.
```

The file *should* exist:

```
inflating: android-x86-release/share/mono-2.0/mono/eglib/eglib-config.h  
```

This implies that perhaps we're extracting into the wrong directory? The [`unzip` command](https://github.com/xamarin/xamarin-android/blob/a10387908dab22e61da2d35418aaf409a14a9ac1/src/mono-runtimes/mono-runtimes.targets#L312) has a working directory of `$(MonoSourceFullPath)\sdks\out`, while this is referencing `$(MonoSourceFullPath)\sdks\builds`; is that correct?